app/controllers/webhooks_controller.rb

```diff
@@ -2,6 +2,9 @@
 
 require 'rest-client'
 
+# TODO(juliexxia):  document
+POOL_DONATION_SELLER_ID = 'send-chinatown-love'
```

Comment:
Can we move this into a constant in the Sellers model?

Reply:
Also could you add this special Seller into our seeds.rb?

Reply:
I made a seller in seeds.rb that I can send over! Might need help with making a new square location for it though (a few of the fields are placeholders such as num_employees, business type, image url <- I left this as shunfa since it won't actually be user facing but we can put another dummy)

`
  {
    seller_id: 'send-chinatown-love',
    cuisine_name: 'Donation Pool',
    name: 'Send Chinatown Love Donation Pool',
    story: "If you can't decide which merchant to donate to, the Donation Pool will split your donation evenly amongst all of our available merchants",
    accept_donations: true,
    sell_gift_cards: true,
    owner_name: 'Send Chinatown Love',
    owner_image_url: 'assets/shunfa-bakery-logo.png',
    target_amount: 1_000_000,
    summary: nil,
    hero_image_url: 'assets/shunfa-bakery-hero.png',
    progress_bar_color: nil,
    business_type: 'Family-owned',
    num_employees: 42,
    founded_year: 2020,
    website_url: nil,
    menu_url: nil,
    square_location_id: 'E4R1NCMHG7B2Y'
  }
`